econml/dml.py

```diff
@@ -208,20 +263,15 @@ class _DMLCateEstimatorBase(_RLearner):
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
-
-    inference: string, inference method, or None
-        Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`).
     """
 
     def __init__(self,
                  model_y, model_t, model_final,
```

Comment:
I would recommend that we add some defaults to these models here too. Most probably non-parametric defaults, e.g.
model_y=RandomForestRegressor()
model_t=RandomForestRegressor() or RandomForestClassifier() dependent on the discrete_treatment flag. 
Maybe we should set: model_t='auto' and internally choose randomforestregressor or randomforestclassifier based on the discrete treatment flag. Even better it would be if we use the XGBRegressor that we are using in the DMLIV prototype, because the XGB regressor is more "autotuned" than the RandomForest of sklearn. 

Similarly in the LinearDMLCateEstimator and SparseLinearDMLCateEstimator, we should choose either LassoCV for model_t or LogisticRegressionCV dependent on the discrete_treatment flag.

Reply:
Using `XGBRegressor` would force us to take a dependency on `xgboost` that we don't currently have (which I would like to avoid unless there's a compelling reason).

Let's discuss your `'auto'` suggestion offline; it's a reasonable suggestion but it seems like something we can do later as part of a separate PR.